lib/render.js

```diff
@@ -26,9 +26,33 @@ function applyRenderHooks(ref, data, locals) {
     return bluebird.resolve(data);
   }
 
-  return bluebird.reduce(setup.plugins, (val, plugin) => {
-    return plugin.render(ref, val, locals);
-  }, data);
+  return bluebird.filter(setup.plugins, (plugin) => plugin.render)
+    .reduce((val, plugin) => {
+      return plugin.render(ref, val, locals);
+    }, data);
+}
+
+/**
+ * Applies `postRender` hooks.
+ * Post Render hooks each returns `html`.
+ *
+ * @param {Object} locals
+ * @param {Object} res
+ * @returns {Function}
+ */
+function applyPostRenderHooks(locals, res) {
+  return (html) => {
+
+    // skip postRender hooks in edit mode
+    if (locals.edit) {
+      return html;
+    }
+
+    return setup.plugins.filter((plugin) => plugin.postRender)
+      .reduce((val, plugin) => {
+        return plugin.postRender(html, res);
```

Comment:
What do you think about passing in `(ref, html, locals)` to match the render pattern? Might be safer to expose locals than to expose the express `res`.

Reply:
The only reason I considered passing res down was because I believe I saw that someone was wanting to be able to add headers or set custom return codes before the `res.send`. But I am fine with not passing it through if you would rather not.